core/templates/core/form_total.html

```diff
@@ -0,0 +1,27 @@
+{% load generic_field %}
+
+<form method="{{ form_method }}"
+      {% if form_id %}id="{{ form_id }}"{% endif %}
+      {% if form_id %}id="{{ form_id }}"{% endif %}
```

Comment:
Why are you adding the id attribute twice?

Reply:
Don't know how that happened. Probably in a copy paste error somewhere, dunno. Fixed it